testsuite/forge-cli/src/main.rs

```diff
@@ -469,9 +470,39 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
             .with_initial_validator_count(NonZeroUsize::new(12).unwrap())
             .with_initial_fullnode_count(12)
             .with_emit_job(EmitJobRequest::default().mode(EmitJobMode::ConstTps { tps: 5000 }))
-            .with_network_tests(vec![&ThreeRegionSimulationTest])
+            .with_network_tests(vec![&ThreeRegionSimulationTest {
```

Comment:
FWIW, I think we should make all these different tests call into a helper function where we define these params. This method is becoming kinda crazy.

Reply:
yeah probably. but I am not going to touch that until  #5490 lands, as that touches SuccessCriteria across all tests